apps/app/pages/_app.js

```diff
@@ -52,6 +52,7 @@ function MyApp({ Component, pageProps }) {
           <GlobalStyle />
           <div>
             <Head>
+              <meta name="google-site-verification" content="KUjgcCuL0UXshh3A0F02itHW6KizSyra4BIsFE9Iz8I" />
```

Comment:
Can we move this to within the `ideas` app? We could put it in the `_app.js` file of our app to achieve the same behavior 

However, since we talked, I got clarification from @JulieMass that this should appear on the Welcome Page only, and not the entire "ideas" app. So we need to go back to your original implementation with that page in /ideas app.